fluffy/deliverables.py

```diff
@@ -13,15 +13,15 @@
 def print_deliverables(output_dir: Path,project_dir: Path, samples: list) -> None:
     """Create a deliverables YAML file"""
 
-    deliverables={"Files":[]}
+    deliverables={"files":[]}
     project_name=str(project_dir).strip("/").split("/")[-1]
```

Comment:
pathlib is excellent at manipulating strings and picking out components like this. Have a look at [this](https://realpython.com/python-pathlib/) guide for example or in std lib. Here you could probably use

```python
project_name = project_dir.parent.name
```